force-app/main/default/classes/DMLOperations.cls

```diff
@@ -145,9 +161,22 @@ public with sharing class DMLOperations {
      * @param oppNames The list of Opportunity names.
      */
     public static void upsertOpportunities(String accountName, List<String> oppNames) {
+        List<Opportunity> newOpps = new List<Opportunity>();
+        Account newAcc = new Account();
+        newAcc.Name = accountName;
+        insert newAcc;
```

Comment:
Here you wanted to check if there was already an existing account by that name first.